.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+.github
```

Comment:
- [ ] Please don't ignore the `.github` file. The `. github` directory houses `workflows` which the github uses to run the files specific to this project like linters etc.
- [ ] According to the project requirements you need to ignore the `test.md `file. Please make sure to  ignore the `test.md `file by adding it to the ` .gitignore `file as it can be added by accident while setting up the linters.